js/src/jupyter-gmaps.js

```diff
@@ -42,16 +61,16 @@ var DirectionsLayerView = GMapsLayerView.extend({
             origin: orig,
             destination: dest,
             waypoints: wps,
-            travelMode: google.maps.DirectionsTravelMode.DRIVING
+            travelMode: google.maps.TravelMode.DRIVING
```

Comment:
Well spotted. It actually looks like both are valid, though the documentation only mentions `TravelMode`, I think:

https://developers.google.com/maps/documentation/javascript/3.exp/reference#TravelMode